### PR TITLE
[bluebinder] Allow use outside script for generate bt address file. Fixes MER#1998

### PR DESCRIPTION
**Pull request by neochapay**

### CODE REVIEW

File: bluebinder_wait.sh [hunk 1]
```diff
@@ -4,6 +4,11 @@ while true
 do
     bt_status=$(/usr/bin/getprop init.svc.bluetooth-1-0)
     if [ "$bt_status" = "running" ] ; then
+#If bt address getted from outside script use it
```
Comment (krnlyng) [inline bluebinder_wait.sh]: nitpick:
Can you indent this on the same line as the above line? with spaces
and maybe replace the comment with something like:
`# If bluetooth address is provided by another script`

File: bluebinder_wait.sh [hunk 2]
```diff
@@ -4,6 +4,11 @@ while true
...
+        if [ -f /var/lib/bluetooth/board-address ] ; then
+            exit 0
+        fi
+#Getting address file from getprop
```
Comment (krnlyng) [inline bluebinder_wait.sh]: also please indent this and put a space after the `#`

Comment (krnlyng) [reply bluebinder_wait.sh]: otherwise lgtm

File: bluebinder_wait.sh (hunk 1 above)
Comment (neochapay) [reply bluebinder_wait.sh]: fixed

**Comment by neochapay** [comment]
@sledges mer-core bugzilla didnt contents combponent bluebinder

**Comment by neochapay** [comment]
@sledges fixed

**Pull request merged**